client/my-sites/sidebar/sidebar.jsx

```diff
@@ -129,11 +129,13 @@ module.exports = React.createClass( {
 		}
 
 		return (
-			<li className={ this.itemLinkClass( '/ads', 'ads' ) }>
-				<a onClick={ this.onNavigate } href={ adsLink }>
-					<span className="menu-link-text">{ site.jetpack ? 'AdControl' : 'WordAds' }</span>
-				</a>
-			</li>
+			<SidebarMenuItem
+				label={ site.jetpack ? 'AdControl' : 'WordAds' }
```

Comment:
Shouldn't this text be translated? Perhaps this could be rewritten as:
```
label={ 
	site.jetpack 
	? this.translate( 'AdControl' )
	: this.translate( 'WordAds' )
}
```